website/addons/gitlab/settings/defaults.py

```diff
@@ -0,0 +1,11 @@
+DEFAULT_HOSTS = ['gitlab.com']
```

Comment:
Is this a valid host? It appears to be more of a sales site than a service site.